### PR TITLE
Doc tweaks for formatting & style + valid layouts on amp-call-tracking

### PR DESCRIPTION
**Pull request by bpaduch**
Reviewed the doc for `amp-call-tracking` and improved styling and wording to be consistent with other reference docs.  Also, added "container" and "nodisplay" as acceptable layouts per what was specified in the [validator-amp-call-tracking](https://github.com/ampproject/amphtml/blob/master/extensions/amp-call-tracking/0.1/validator-amp-call-tracking.protoascii) rules.

to: @alanorozco 
cc: @jasti 

related to: https://github.com/ampproject/amphtml/pull/7493

### CODE REVIEW

File: extensions/amp-call-tracking/amp-call-tracking.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ limitations under the License.
   </tr>
   <tr>
     <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
-    <td>fill, fixed, fixed-height, flex-item, responsive</td>
+    <td>container, fill, fixed, fixed-height, flex-item, nodisplay, responsive</td>
```
Comment (alanorozco) [inline extensions/amp-call-tracking/amp-call-tracking.md]: Original set of values was correct.

Comment (bpaduch) [inline extensions/amp-call-tracking/amp-call-tracking.md]: Hmm, okay, but what does the [validator](https://github.com/ampproject/amphtml/blob/master/extensions/amp-call-tracking/0.1/validator-amp-call-tracking.protoascii#L73) for amp-call-tracking have more layouts?

```
    supported_layouts: CONTAINER
    supported_layouts: FILL
    supported_layouts: FIXED
    supported_layouts: FIXED_HEIGHT
    supported_layouts: FLEX_ITEM
    supported_layouts: NODISPLAY
    supported_layouts: RESPONSIVE
```

Comment (alanorozco) [inline extensions/amp-call-tracking/amp-call-tracking.md]: The validator proto and the JS implementation are inconsistent. That was a mistake on my part.  Created #7732 to fix it.

Comment (alanorozco) [inline extensions/amp-call-tracking/amp-call-tracking.md]: ...and actually `CONTAINER` should be valid. Updated #7732 to reflect that. 

In summary, the valid set is **container, fill, fixed, fixed-height, flex-item, responsive**.

**Comment by bpaduch** [comment]
Removed `nodisplay`.

**Pull request merged**